Cargo.toml

```diff
@@ -56,7 +57,7 @@ compact_filters = ["rocksdb", "socks", "lazy_static", "cc"]
 key-value-db = ["sled"]
 async-interface = ["async-trait"]
 all-keys = ["keys-bip39"]
-keys-bip39 = ["tiny-bip39"]
+keys-bip39 = ["tiny-bip39", "zeroize"]
```

Comment:
it seems `all-keys` and `keys-bip39` are duplicated feature. Is this intentional?

Reply:
I think the idea is we eventually want to get away from using tiny-bip39, so want to use a feature name that won't change.

Reply:
But for all-keys I think it's just  to be prepared if we want to have a separate optional key subfeatures or the all-keys feature.

Reply:
Makes Sense. Thanks for clarifying.